### PR TITLE
fixes #3060, fixes #3186

### LINKED ISSUE #3060: Error when editing motion comments in config
When adding/removing a motion comment field in the config, I get a `Error: "id" must be a string or a number!` thrown by the openslides-libs. This is not a major problem, because as I can see everything works fine. My guess: The green check-sign does not appear on add/remove, so this can be the problem.

### LINKED ISSUE #3186: Missing request for motion comments in config
If you click on 'Add motion comment' no request is send --> reload page and the field is gone. If you change the name or public/private result in a request. Also removing of fields does not work for me.

### PR DESCRIPTION
**Pull request by FinnStutzenstein**

### CODE REVIEW

**Comment by normanjaeckel** [comment]
Please add CHANGELOG entry.

**Comment by FinnStutzenstein** [comment]
@normanjaeckel Done.

File: CHANGELOG [hunk 1]
```diff
@@ -7,6 +7,8 @@ https://openslides.org/
 Version 2.1.2 (unreleased)
 ==========================
 
+- Fixed issue when creating/deleting motion comment fields in the settings.
```
Comment (normanjaeckel) [inline CHANGELOG]: Either we change this entry or we change the branch to `stable/2.1.x`.

Comment (emanuelschuetze) [inline CHANGELOG]: Please change "2.1.2" to "2.2" in CHANGELOG. All new pull requests are scheduled for 2.2...

Comment (FinnStutzenstein) [inline CHANGELOG]: Done.

**Comment by FinnStutzenstein** [comment]
@emanuelschuetze Done.

**Comment by emanuelschuetze** [comment]
Rebased and resolved changelog conflict.

**Pull request merged**